src/api/src/main/java/engineering/everest/lhotse/api/config/SwaggerConfig.java

```diff
@@ -1,121 +1,120 @@
 import springfox.documentation.spi.DocumentationType;
-import springfox.documentation.spi.service.contexts.OperationContext;
 import springfox.documentation.spi.service.contexts.SecurityContext;
 import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger.web.SecurityConfiguration;
+import springfox.documentation.swagger.web.SecurityConfigurationBuilder;
 
-import java.util.List;
-import java.util.Set;
-import java.util.function.Predicate;
-
-import static java.util.Arrays.asList;
-import static java.util.Arrays.stream;
-import static java.util.stream.Collectors.toList;
-import static java.util.stream.Stream.concat;
 import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
 import static springfox.documentation.builders.PathSelectors.regex;
 
+import java.util.Arrays;
+import java.util.Set;
+import java.util.function.Predicate;
+
 @Profile("!prod")
 @Configuration
 @ComponentScan({"engineering.everest.lhotse.api"})
 @PropertySource("swagger.properties")
 @SwaggerDefinition(tags = {
-        @Tag(name = "System", description = "Information on the system you are interacting with"),
-        @Tag(name = "Organizations", description = "Organization information and management"),
-        @Tag(name = "Users", description = "User information and management"),
-        @Tag(name = "OrgAdmins", description = "Organization Admin information and management"),
+    @Tag(name = "System", description = "Information on the system you are interacting with"),
+    @Tag(name = "Organizations", description = "Organization information and management"),
+    @Tag(name = "Users", description = "User information and management"),
+    @Tag(name = "OrgAdmins", description = "Organization Admin information and management"),
 })
 public class SwaggerConfig {
-    private final List<String> anonymousUserAntPaths;
-    private final List<String> authenticatedAntPaths;
-    private final AntPathMatcher antPathMatcher;
-
-    @SuppressWarnings("PMD.UseVarargs")
-    public SwaggerConfig(
-            @Value("${application.security.endpoint.matchers.anonymous}") String[] anonymousUserAntPaths,
-            @Value("${application.security.endpoint.matchers.authenticated}") String[] authenticatedAntPaths,
-            @Value("${application.security.endpoint.matchers.admin}") String[] adminUserAntPaths) {
-        this.anonymousUserAntPaths = asList(anonymousUserAntPaths);
-        this.authenticatedAntPaths = concat(stream(adminUserAntPaths), stream(authenticatedAntPaths))
-                .collect(toList());
-        this.antPathMatcher = new AntPathMatcher();
-    }
+    @Value("${keycloak.auth-server-url}")
+    private String authServer;
+   
+    @Value("${keycloak.credentials.secret}")
+    private String clientSecret;
+   
+    @Value("${keycloak.resource}")
+    private String cliendId;
+   
+    @Value("${keycloak.realm}")
+    private String realm;
 
     @Bean
     public Docket apiDocumentation() {
```

Comment:
I noticed that the '/api/version' endpoint is showing up as protected in Swagger when it is public.

Reply:
Investigating an issue may take time.
Created a card in project dashboard to track it later.